deny.toml

```diff
@@ -58,6 +58,7 @@ allow = [
     "LicenseRef-ring",
     "LicenseRef-webpki",
     "WTFPL",
+    "BSL-1.0",
```

Comment:
Boost?

Reply:
Yep... this one library uses it, and it's a very permissive license. I don't believe it's an issue. Is it?

Reply:
Nope not an issue at all, there are just some license with similar initials which aren't so friendly so wanted to double check